controllers/topology/internal/mergepatch/mergepatch.go

```diff
@@ -246,7 +246,18 @@ func enforcePath(authoritative, twoWay map[string]interface{}, path contract.Pat
 		return
 	case 1:
 		// If we are at the end of a path, enforce the value.
-		twoWay[path[0]] = authoritative[path[0]]
+
+		// If there is an authoritative change for the value, apply it.
+		if authoritativeChange, authoritativeHasChange := authoritative[path[0]]; authoritativeHasChange {
+			twoWay[path[0]] = authoritativeChange
+			return
+		}
+
+		// Else, if there is no authoritative change but there is a twoWays change for the value, blank it out.
+		if _, twoWayHasChange := twoWay[path[0]]; twoWayHasChange {
+			delete(twoWay, path[0])
+		}
```

Comment:
```suggestion
		delete(twoWay, path[0])
```
I think that's what the linter is telling us